tests/fast-integration/eventing-test/eventing-test.js

```diff
@@ -99,6 +112,11 @@ describe('Eventing tests', function() {
 
   // eventingTestSuite - Runs Eventing tests
   function eventingTestSuite(backend, isSKR, testCompassFlow=false) {
+    // This code can be removed when we have release 2.10
+    it('redeploys the function to use the latest code', async function() {
+      await redeployFunction(testNamespace);
```

Comment:
Can we confirm if we still need to redeploy the lastorder func?

Reply:
we dont need to redeploy